packages/cascara/src/ui/Table/TableFooter.js

```diff
@@ -1,84 +1,53 @@
 import React, { useContext } from 'react';
 
-import Button from '../Button';
-import DownloadButton from '../../modules/DownloadButton';
+import { ModuleContext } from '../../modules/context';
 
-import TableContext from './context';
-import SelectionToggle from './atoms/SelectionToggle';
 import ActionBar from './ActionBar';
+import ActionButton from '../../modules/ActionButton';
+import ActionEdit from '../../modules/ActionEdit';
+import DownloadButton from '../../modules/DownloadButton';
 
 const TableFooter = () => {
-  const {
-    actions,
-    bulkActions,
-    dataConfig,
-    handleOnAction,
-    selection,
-    selectionIsEnabled,
-  } = useContext(TableContext);
-  const selectionCell = (
-    <th key={'main-toggle'}>
-      <SelectionToggle id={'__ALL__'} />
-    </th>
+  const { dataConfig } = useContext(ModuleContext);
+
+  const actionBarCell = (
+    <ActionBar
+      actions={dataConfig.actions.map((action) => {
+        let Module;
+
+        switch (action.module) {
+          case 'edit':
+            Module = ActionEdit;
```

Comment:
This is the case where we need to define a better way for mapping `dataType` -> `Module`.